newton/_src/sensors/tiled_camera_sensor.py

```
@@ -0,0 +1,314 @@
+            dim=(self.render_context.num_cameras, self.render_context.height, self.render_context.width),
+            inputs=[
+                self.render_context.width,
+                self.render_context.height,
+                camera_fovs,
+                self.render_context.camera_rays,
+            ],
+        )
+
+    def save_color_image(self, filename: str) -> bool:
+        try:
+            from PIL import Image  # noqa: PLC0415
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)